src/layouts/Media/Media.tsx

```diff
@@ -305,14 +351,16 @@ export const Media = (): JSX.Element => {
       />
 
       <SiteEditLayout overflow="hidden">
-        <VStack spacing="2rem" w="100%">
+        <VStack spacing={0} w="100%">
           {/* Header section */}
           <HStack w="100%">
             {/* Page title segment */}
             <VStack spacing="0.5rem" align="left" w="40%">
               <MediaBreadcrumbs />
               <Text as="h4" textStyle="h4" mt="0.75rem">
-                All {mediaType}
+                {mediaType === mediaDirectoryName
+                  ? `All ${mediaType}`
+                  : _.upperFirst(mediaDirectoryName.split("%2F").pop())}
```

Comment:
could we do a `.get` as this alters `mediaDirectoryName`

Reply:
Hmm but we do a `split()` first so that would create a new copy as an array.